docker/buildDocker.sh

```diff
@@ -99,51 +99,46 @@ checkArgs()
         	echo "WORKSPACE not found, setting it as environment variable 'HOME'"
         	WORKSPACE=$HOME
 	fi
-	if [[ "$useEclipseDockerFiles" == true && "$useEclipseDockerSlavesFiles" == true ]]; then
-		echo "Unable to use both kinds of dockerfiles at once, Select a single option."
-		exit 1
-        fi
+	if [ "$cleanWorkspace" == true ]; then
+		echo "Cleaning workspace"
+		rm -rf $WORKSPACE/DockerBuildFolder/$jdkVersion-$buildVariant
+	fi
 }
 
 useEclipseDockerFiles()
 {
-	cd $WORKSPACE/DockerBuildFolder/$jdkVersion-$buildVariant && mkdir -p EclipseDockerfiles
-	cd EclipseDockerfiles
-	for jdk in $jdkVersion
-	do
-		# ${jdk%?} will remove the 'u' from 'jdk__u' when needed.
-		curl -o Dockerfile.$jdk https://raw.githubusercontent.com/eclipse/openj9/master/buildenv/docker/${jdk%?}/x86_64/ubuntu16/Dockerfile;
-		sharedEclipseDockerCommands $jdk
-	done
-}
-
-useEclipseDockerSlavesFiles()
-{
-	cd $WORKSPACE/DockerBuildFolder/
-	git clone https://github.com/eclipse/openj9 $WORKSPACE/DockerBuildFolder/$jdkVersion-$buildVariant 
-	cd $WORKSPACE/DockerBuildFolder/$jdkVersion-$buildVariant/buildenv/jenkins/docker-slaves/x86/centos6.9
-	if [ -f "known_hosts" ]; then 
-		rm known_hosts
-	fi
+	mkdir -p $WORKSPACE/DockerBuildFolder/$jdkVersion-$buildVariant/EclipseDockerfiles
+	cd $WORKSPACE/DockerBuildFolder/$jdkVersion-$buildVariant/EclipseDockerfiles
+	wget https://raw.githubusercontent.com/eclipse/openj9/master/buildenv/docker/mkdocker.sh
+	chmod +x mkdocker.sh
+	# Generate an Ubuntu1804 Dockerfile using mkdocker.sh
+	$WORKSPACE/DockerBuildFolder/$jdkVersion-$buildVariant/EclipseDockerfiles/mkdocker.sh --dist=ubuntu --version=18 --print >> $WORKSPACE/DockerBuildFolder/$jdkVersion-$buildVariant/EclipseDockerfiles/Dockerfile
+	
+	# This Dockerfile requires an ssh key, authorized_key and known_hosts file to build
+	ssh-keygen -q -f id_rsa -t rsa -N ''
+	cat id_rsa.pub >> authorized_keys
 	ssh-keyscan github.com >> $PWD/known_hosts
-	cp $HOME/.ssh/id_rsa.pub $PWD
-	mv id_rsa.pub authorized_keys
+
+	# ${jdk%?} will remove the 'u' from 'jdkXXu'.
 	for jdk in $jdkVersion
 	do
-		cp Dockerfile $PWD/Dockerfile.$jdk
-		sharedEclipseDockerCommands $jdk
+		if [ ${jdk} != "jdk" ]; then
+			jdk=${jdk%?}
+		fi
+		manualDockerCommands ${jdk}
 	done
 }
 
-sharedEclipseDockerCommands()
+manualDockerCommands()
```

Comment:
Are they manual commands?  Looks like we're running them automated...

Reply:
haha, good point. I'll rename it- by 'manual', I meant 'not using makejdk-any-platform to create the docker container'.